api/challenge_api/.gitignore

```diff
@@ -0,0 +1,16 @@
+/.bundle
+
+/db/*.sqlite3
+/db/*.sqlite3-journal
```

Comment:
I don't believe we'll be using SQLite for anything unless Rails uses it internally for something? We'll eventually be using Mongo, but we can worry about adding that in a later commit.

Reply:
Totally forgot about taking out SQLite, my mistake! I have the mongo gem in the Gemfile but haven't configured it any further.